cmake/std/PullRequestLinuxCuda10.1.243TestingSettings.cmake

```diff
@@ -129,11 +129,10 @@ set (KokkosCore_UnitTest_Cuda_MPI_1_EXTRA_ARGS
   "--gtest_filter=-cuda.debug_pin_um_to_host:cuda.debug_serial_execution"
   CACHE STRING "Temporary disable for CUDA PR testing")
 
-# Disable a few failing tests for initial release of Cuda 10.1.105 PR build
-# set (EpetraExt_inout_test_LL_MPI_4_DISABLE ON CACHE BOOL "Temporary disable for CUDA PR testing")
-# set (EpetraExt_inout_test_MPI_4_DISABLE ON CACHE BOOL "Temporary disable for CUDA PR testing")
-# set (Teko_testdriver_MPI_4_DISABLE ON CACHE BOOL "Temporary disable for CUDA PR testing")
-# set (Zoltan2_fix4785_MPI_4_DISABLE ON CACHE BOOL "Temporary disable for CUDA PR testing")
+# Disable a few failing tests for initial release of Cuda 10.1.243 PR build
```

Comment:
A comment here about why they are disabled would be good, so that when the file propagates to some future build on some other system (or SpectrumMPI gets fixed), they might be re-enabled.
Thanks, @prwolfe